draft-ietf-webpush-protocol.xml

```diff
@@ -1137,6 +1147,12 @@ HEADERS      [stream 82] +END_STREAM
           This could force the user agent to serialize subscription requests to the push
           service.
         </t>
+        <t>
+         A user agent that monitors subscriptions for other clients might

```

Comment:
I was referring to "Moving between UA" as tricky. 

My reading of the previous version is that it allows UA to handle multiple sets - like the
multi-user case in android.